### PR TITLE
Feature/preparation for importance

### PR DESCRIPTION
**Pull request by knordstrom**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ processor = "processor.main:start"
 event_processor = "processor.main:start_kafka_calendar"
 document_processor = "processor.main:start_kafka_documents"
 employee_processor = "employees.main:start"
+slack_processor = "slack.main:start"
```
Comment (mshash8) [inline pyproject.toml:15]: What does adding line 14 help us do?

Comment (knordstrom) [reply pyproject.toml:15]: `poetry run slack_processor`

Comment (mshash8) [reply pyproject.toml:15]: Okay perfect, thanks!

**Review by mshash8 — APPROVED** [review]
LGTM!

**Pull request merged**